Answers.md

```diff
@@ -1,11 +1,17 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+React is a JavaSCript library for Building UI's. It is designed to be able to handle large amounts of data to be interfaced with while maximizing performance. 
 
 1.  What does it mean to _think_ in react?
+It means to break down each aspect of a website into components and subcomponents and think about how they will interact through states and props.
+
 
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+A functional/presentational component is designed to display static information on the webopage. Class/Stateful components are designed to display information that can be altered and re-rendered. 

```

Comment:
the information isn't necessarily static, it's just coming from props and controlled in a more dynamic component